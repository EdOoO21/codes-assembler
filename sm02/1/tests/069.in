-924 -804 641 -659 -864 239 -295 -329 883 873 788 613 627 770 124 -152 -49 697 -123 161 -104 701 63 -343 146 122 656 -99 -710 -156 -197 -869 -152 -414 -699 -599 -709 -365 -568 420 883 353 919 546 901 485 30 672 -575 -789 -797 -207 821 273 492 -867 -464 787 636 480 243 -238 820 -917 -872 -808 -553 -938 -703 -490 834 -500 -440 -692 840 -20 283 644 -836 437 -328 297 -461 906 95 61 -147
