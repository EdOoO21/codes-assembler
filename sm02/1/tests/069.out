-147
61
95
906
-461
297
-328
437
-836
644
283
-20
840
-692
-440
-500
834
-490
-703
-938
-553
-808
-872
-917
820
-238
243
480
636
787
-464
-867
492
273
821
-207
-797
-789
-575
672
30
485
901
546
919
353
883
420
-568
-365
-709
-599
-699
-414
-152
-869
-197
-156
-710
-99
656
122
146
-343
63
701
-104
161
-123
697
-49
-152
124
770
627
613
788
873
883
-329
-295
239
-864
-659
641
-804
-924
